src/components/GlobalErrorBoundary/index.tsx

```diff
@@ -43,13 +43,29 @@ const LinkContent = styled.div`
   }
 `
 
-type Props = {
-  error: Error
-  componentStack: string | null
-  resetError: () => void
-}
+const GlobalErrorBoundaryFallback: FallbackRender = ({ error, componentStack }) => {
+  //  When loading app during release, chunk load failure may occur
+  const chunkFailedMessage = /Loading chunk [\d]+ failed/
+  const isChunkError = error?.message && chunkFailedMessage.test(error.message)
+
+  const time = new Date().getTime()
+
+  const lastReloadKey = 'lastReload'
+  const lastReloadString = localStorage.getItem(lastReloadKey)
+  const lastReload = lastReloadString && JSON.parse(lastReloadString)
+
+  // Expires 10 secs after last chunk failure reload
+  const isExpired = time > +lastReload + 10000
```

Comment:
`10000` -> `const MIN_RELOAD_TIME = 10e3`

Reply:
I'd argue 10e3 is more readable than 10000 😄

Reply:
My point is the const.